### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Xueying-Huyan**

### CODE REVIEW

**Comment by guiguem** [comment]
Hi @Xueying-Huyan
 
Thanks for this PR!
I am working on fixing this dockerhub issue. Hopefully this should not take long...

For the release, there are some missing pieces: 
- Dockerfile version
- validation log update
- CmakeLists.txt version bump

We can have the review here, but maybe it would be easier for the actual merge to do this using git-flow?
This would do the tagging and proper merge a bit simpler...

**Comment by guiguem** [comment]
It seems the dockerhub issue was fixed by my last commit; some obscure way docker does its pull: https://forums.docker.com/t/automated-builds-with-submodules/33797

File: documentation/validation_log.rst [hunk 1]
```diff
@@ -4,6 +4,20 @@ Validation Log
...
+Version: v
+~~~~~~~~~~~~~~~~
+
+Release Date: 
+'''''''''''''''''''''''''''''
+
+New features:
+'''''''''''''
+
+* morpho as git submodule
```
Comment (guiguem) [inline documentation/validation_log.rst:17]: Here there should be more informations about what this release is about: so much work done by all the SA group!!!!!!

Comment (Xueying-Huyan) [reply documentation/validation_log.rst]: Thank you for your comments, Mathieu!  I just added more features to the validation_log.rst file and updated the mermithid version in Dockerfile and CmakeLists.txt.

**Comment by nsoblath** [comment]
This merge was done last week.

**Pull request closed without merging**